app/forms/provider_interface/offer_wizard.rb

```diff
@@ -148,5 +183,22 @@ def update_path_history(attrs)
       @wizard_path_history.update
       @path_history = @wizard_path_history.path_history
     end
+
+    def setup_further_conditions
+      self.further_conditions = 4.times.map do |index|
```

Comment:
Can we extract `4` into a constant, so it has some context. Maybe `MAX_NUMBER_OF_CONDITIONS_ALLOWED` (or something shorter)?

Reply:
Gone with `NUMBER_OF_FURTHER_CONDITIONS` because it is actually a fixed number of conditions at the moment. (To be changed later when we allow add another)